www/docusaurus.config.ts

```
@@ -190,6 +193,315 @@ export default {
+                }
+                if (
+                  docItem.id.startsWith('typedoc/') ||
+                  docItem.id.includes('/typedoc/')
+                ) {
+                  return false;
+                }
+                return true;
+              })
+              .map((docItem) => {
+                let link = `${versionSpec.routePath}/${docItem.id}`;
+                link = link.replace(new RegExp('([^:])//', 'g'), '$1/');
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)